### DISCUSSION TITLE
Paste website -- Combobox -- Combobox with custom group label -- Error

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#1011] [Q&A] [answered]
https://paste.twilio.design/components/combobox/#gatsby-focus-wrapper

Combobox with custom group label -- example gives following error.
TypeError: Cannot read property 'toLowerCase' of null

Attached screencapture for more details.
<img width="578" alt="Screen Shot 2020-12-07 at 9 29 26 PM" src="https://user-images.githubusercontent.com/69016180/101443939-50377a00-38d3-11eb-98f8-8b4a67cc36c6.png">

**Comment by TheSisb** [comment]
Thanks for reporting this! Added it to our backlog for fixing.

**Comment by serifluous** [comment, marked as answer]
Thanks, @ishah-twilio ! The example is fixed now: https://paste.twilio.design/components/combobox/#combobox-with-custom-group-label
